sql/understanding-the-query-execution-plan.md

```diff
@@ -29,6 +29,45 @@ Currently, the `EXPLAIN` statement returns the following four columns: id, count
 | task | the task that the current operator belongs to. The current execution plan contains two types of tasks: 1) the **root** task that runs on the TiDB server; 2) the **cop** task that runs concurrently on the TiKV server. The topological relations of the current execution plan in the task level is that a root task can be followed by many cop tasks. The root task uses the output of cop task as the input. The cop task executes the tasks that TiDB pushes to TiKV. Each cop task scatters in the TiKV cluster and is executed by multiple processes. |
 | operator info | The details about each operator. The information of each operator differs from others, see [Operator Info](#operator-info).|
 
+### Example Usage
+
+Using the [bikeshare example database](../bikeshare-example-database.md):
+
+```
+mysql> EXPLAIN SELECT count(*) FROM trips WHERE start_date BETWEEN '2017-07-01 00:00:00' AND '2017-07-01 23:59:59';
++--------------------------+-------------+------+------------------------------------------------------------------------------------------------------------------------+
+| id                       | count       | task | operator info                                                                                                          |
++--------------------------+-------------+------+------------------------------------------------------------------------------------------------------------------------+
+| StreamAgg_20             | 1.00        | root | funcs:count(col_0)                                                                                                     |
+| └─TableReader_21         | 1.00        | root | data:StreamAgg_9                                                                                                       |
+|   └─StreamAgg_9          | 1.00        | cop  | funcs:count(1)                                                                                                         |
+|     └─Selection_19       | 8166.73     | cop  | ge(bikeshare.trips.start_date, 2017-07-01 00:00:00.000000), le(bikeshare.trips.start_date, 2017-07-01 23:59:59.000000) |
+|       └─TableScan_18     | 19117643.00 | cop  | table:trips, range:[-inf,+inf], keep order:false                                                                       |
++--------------------------+-------------+------+------------------------------------------------------------------------------------------------------------------------+
+5 rows in set (0.00 sec)
+```
+
+Here we can see that the coprocesor (cop) needs to scan the table `trips` to find rows that match the criteria of `start_date`. Rows that meet this criteria are determined in `Selection_19` and passed to `StreamAgg_9`, all still within the coprocessor (i.e. inside of TiKV). Each of the TiKV nodes return `1.00` row to TiDB (as `TableReader_21`), which are then aggregated as `StreamAgg_20` to return `1.00` row to the client.
```

Comment:
should we clearly specify that the value in the `count` column, for example `1.00`, are estimated with the help of table statistics?